release-notes/2024-04/README.md

```diff
@@ -0,0 +1,203 @@
+# SystemLink Enterprise release 2024-04 Release Notes
+
+The 2024-04 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- `feedservice 0.1.211`
+    - New helm chart added for feeds. The following existing privileges are being used:
+        - feed:Read
+        - feed:Create
+        - feed:Update
+        - feed:Delete
+- `feedsui 0.0.364`
+    - New helm chart added for feeds.
+- `serviceregistry 0.13.12`
+    - Added new `testplan:Execute` permission to allow calling the new `/niworkorder/v1/testplans/{testPlanId}/execute` route. The Work Order service is not installed by default. This route is marked "[Beta]" and is still in development.
+
+## Helm Chart Breaking Changes
+
+- `workorder 0.3.79`
+    - workorder database needs to be dropped on upgrade.
```

Comment:
How does a customer do this. What is our recommended procedure for doing so. If this process is long we should ask the team responsible for work-order to draft a doc on how to do this and reference it from here.

Reply:
Basically they stop the work order deployment, log in to the mongo pod, and use mongosh commands to drop the db. I will push a document w/ instructions and a link to this branch later today.

Reply:
I pushed a draft and added a link.